app/src/main/java/ch/hikemate/app/ui/map/HikeDetailScreen.kt

```diff
@@ -93,17 +96,18 @@ object HikeDetailScreen {
   const val MAP_MIN_LONGITUDE = -180.0
   const val MAP_BOUNDS_MARGIN: Int = 100
 
-  const val TEST_TAG_MAP = "HikeDetailScreenMap"
-  const val TEST_TAG_HIKE_NAME = "HikeDetailScreenHikeName"
-  const val TEST_TAG_BOOKMARK_ICON = "HikeDetailScreenBookmarkIcon"
-  const val TEST_TAG_ELEVATION_GRAPH = "HikeDetailScreenElevationGraph"
-  const val TEST_TAG_DETAIL_ROW_TAG = "HikeDetailScreenDetailRowTag"
-  const val TEST_TAG_DETAIL_ROW_VALUE = "HikeDetailScreenDetailRowValue"
-  const val TEST_TAG_ADD_DATE_BUTTON = "HikeDetailScreenAddDateButton"
-  const val TEST_TAG_PLANNED_DATE_TEXT_BOX = "HikeDetailScreenPlannedDateTextBox"
-  const val TEST_TAG_DATE_PICKER = "HikeDetailDatePicker"
-  const val TEST_TAG_DATE_PICKER_CANCEL_BUTTON = "HikeDetailDatePickerCancelButton"
-  const val TEST_TAG_DATE_PICKER_CONFIRM_BUTTON = "HikeDetailDatePickerConfirmButton"
+  const val HIKE_DETAILS_TEST_TAG_MAP = "HikeDetailScreenMap"
+  const val HIKE_DETAILS_TEST_TAG_HIKE_NAME = "HikeDetailScreenHikeName"
+  const val HIKE_DETAILS_TEST_TAG_BOOKMARK_ICON = "HikeDetailScreenBookmarkIcon"
+  const val HIKE_DETAILS_TEST_TAG_ELEVATION_GRAPH = "HikeDetailScreenElevationGraph"
+  const val HIKE_DETAILS_TEST_TAG_DETAIL_ROW_TAG = "HikeDetailScreenDetailRowTag"
+  const val HIKE_DETAILS_TEST_TAG_DETAIL_ROW_VALUE = "HikeDetailScreenDetailRowValue"
+  const val HIKE_DETAILS_TEST_TAG_ADD_DATE_BUTTON = "HikeDetailScreenAddDateButton"
+  const val HIKE_DETAILS_TEST_TAG_PLANNED_DATE_TEXT_BOX = "HikeDetailScreenPlannedDateTextBox"
+  const val HIKE_DETAILS_TEST_TAG_DATE_PICKER = "HikeDetailDatePicker"
+  const val HIKE_DETAILS_TEST_TAG_DATE_PICKER_CANCEL_BUTTON = "HikeDetailDatePickerCancelButton"
+  const val HIKE_DETAILS_TEST_TAG_DATE_PICKER_CONFIRM_BUTTON = "HikeDetailDatePickerConfirmButton"
+  const val HIKE_DETAILS_TEST_TAG_RUN_HIKE_BUTTON = "HikeDetailRunHikeButton"
```

Comment:
Question: Wasn't the convention more about the value of the tag and not the variable name of it ?

Reply:
test

Reply:
Thanks!